tests/Unit/PointwiseFunctions/Hydro/TestFunctions.py

```diff
@@ -3,6 +3,26 @@
 
 import numpy as np
 
+
+# Functions for testing ComovingMagneticField.cpp
```

Comment:
The rest of the file has 2 blank lines after these "opening` comments. Could you please do the same here?

Reply:
I guess just move the blank lines into the previous commit :)